ui/ui-enterprise/pages/app/new/index.tsx

```diff
@@ -60,6 +62,14 @@ export default function Page(props) {
 
 Page.getInitialProps = async (ctx) => {
   const { execApi } = UseApi(ctx);
-  const { plugins }: Actions.AppOptions = await execApi("get", `/appOptions`);
-  return { plugins };
+  const { plugins }: Actions.PluginsList = await execApi("get", `/plugins`, {
+    includeInstalled: true,
+    includeAvailable: false,
+    includeVersions: false,
+  });
+  return {
+    plugins: plugins
+      .filter((p) => p.apps?.length > 0)
+      .sort((a, b) => (a.name > b.name ? 1 : -1)),
```

Comment:
Since we are now loading all plugins, we only want to display those which have apps (not `@grouparoo/sentry`, for example)